src/main/java/carpet/script/api/Auxiliary.java

```diff
@@ -451,7 +451,7 @@ else if (!interactable && targetBlock == null)
             {
                 ServerPlayerEntity player = EntityValue.getPlayerByValue(s.getMinecraftServer(), res);
                 if (player == null) return LazyValue.NULL;
-                s = player.getCommandSource();
+                playersVal = res;
```

Comment:
??

Reply:
No, cos it could be a list of players, so I need to shift that check, just realised.

Reply:
wait IDEA didnt tell me that u removed that var's initialization

Reply:
ocrap that whole thing got deleted.